manifests/init.pp

```diff
@@ -63,6 +63,7 @@
   $watches           = {},
   $checks            = {},
   $acls              = {},
+  $umask             = 0022,
```

Comment:
It would be a tad safer to put this in quotes as it is meant to be treated as a string